Frends.Instagram.Request/Frends.Instagram.Request/Frends.Instagram.Request.cs

```diff
@@ -0,0 +1,154 @@
+﻿#pragma warning disable SA1200 //Using directives should be placed correctly.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Frends.Instagram.Request.Definitions;
+
+namespace Frends.Instagram.Request;
+
+/// <summary>
+/// Facebook class.
+/// </summary>
+public static class Instagram
+{
+    /// <summary>
+    /// This is task for reading data from Instagram API.
+    /// [Documentation](https://tasks.frends.com/tasks/frends-tasks/Frends.Facebook.Get).
+    /// </summary>
+    /// <param name="input">Set reference type, parameters and token.</param>
+    /// <param name="options">Optional parameters.</param>
+    /// <param name="cancellationToken">Cancellation token given by Frends.</param>
+    /// <returns>Object { int StatusCode, dynamic Message }.</returns>
+    public static async Task<Result> Request([PropertyTab] Input input, [PropertyTab] Options options, CancellationToken cancellationToken)
+    {
+        if (string.IsNullOrEmpty(input.AccessToken))
+            throw new ArgumentNullException(nameof(input.AccessToken) + " cannot be empty.");
+        else if (string.IsNullOrEmpty(input.ApiVersion))
```

Comment:
Please use singular if statements instead of else if when you throw exception from them.